docs/UserGuide.md

```diff
@@ -120,17 +120,29 @@ Examples:
 *  `edit 2 pn/AB12345J pe/31-12-2024` updates the policy at the INDEX number 2 with the new policy number and expiration date.
 
 Acceptable values for each parameter:
-* `l/LICENCEPLATE`: Alphanumeric, _up to_ 9 characters.
+* `n/NAME`: Alphabets.
+* `i/NRIC`: Alphanumeric, _exactly_ 4 characters.
+* `p/CONTACT NUMBER`: Numeric, _exactly_ 8 characters.
+* `e/EMAIL`: Alphanumeric /and special characters, no white spaces allowed, standard email format.
+* `t/TAG`: Alphabets, no white spaces allowed.
+* `c/COMPANY`: Alphabets and/or special characters, white spaces allowed.
+* `l/LICENCE PLATE`: Alphanumeric, _up to_ 9 characters.
 * `pn/POLICY NUMBER`: Alphanumeric, _exactly_ 8 characters.
 * `pi/POLICY ISSUE DATE` and `pe/POLICY EXPIRY DATE`: Date in the format dd-mm-yyyy.
 
-Expected output upon success: [coming soon]
+Expected output upon success: <br>
+![EditSuccess](images/EditSuccess.png)
```

Comment:
Image provided is able to show the users exactly what it will look like on success, very nice!